pkg/daemon/discover/discover.go

```diff
@@ -211,15 +211,16 @@ func udevBlockMonitor(c chan string, period time.Duration) {
 		[]string{"(?i)add", "(?i)remove"},
 		udevFilter)
 
+	ticker := time.NewTicker(period)
```

Comment:
shouldn't this be initialized above the second for loop (where `NewTimer` was initialized)? 
In other words, won't the `ticker.C` expire as soon as we enter the second for loop as the first for loop waits for the `events`.

Reply:
maybe we no need to wait `ticker.C` when first hotplug event fires, we only need to handle batching them up together and wait `ticker.C` before the previous one is done.

Reply:
The existing `NewTimer` is apt for this operation instead of a Ticker, IMO.